qlog/qlog_dir.go

```diff
@@ -0,0 +1,58 @@
+package qlog
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/quic-go/quic-go/internal/utils"
+	"github.com/quic-go/quic-go/logging"
+)
+
+// QlogDir contains the value of the QLOGDIR environment variable.
+// If it is the empty string ("") no qlog output is written.
+var QlogDir string
```

Comment:
Why is this exported?

Reply:
It is exposed to programmatically change it :)